tempesta_fw/http_parser.c

```diff
@@ -1592,8 +1602,10 @@ tfw_http_parse_req(void *req_data, unsigned char *data, size_t len)
 
 	/* Request headers are fully read. */
 	__FSM_STATE(Req_HdrDone) {
-		if (c == '\n')
+		if (c == '\n') {
+			__field_finish_curpos(msg, &req->crlf, data, p);
```

Comment:
There is no sense to introduce the new function of 2 lines and called only once.
Please see comment in #326 - probably we should set crlf.len by 2, however setting TFW_STR_COMPLETE is appreciated as well...